src/pages/Swap/index.tsx

```diff
@@ -383,7 +383,11 @@ export default function Swap({ history }: RouteComponentProps) {
                 <AutoColumn gap="8px" style={{ padding: '0 16px' }}>
                   {Boolean(trade) && (
                     <RowBetween align="center">
-                      <Text fontWeight={500} fontSize={14} color={theme.text2}>
+                      <Text
+                        fontWeight={500}
+                        fontSize={14}
+                        // color={theme.text2}
```

Comment:
Disabling the color prop.

Reply:
you need to rework this commented color prop as it's affecting way too many files and these aren't modded files either